packages/editor/src/components/parts/mail/MailAttachmentPart.tsx

```diff
@@ -46,16 +48,15 @@ const MailAttachmentTable = () => {
   );
 
   const onChange = (change: MailAttachment[]) => {
-    setData(change);
```

Comment:
This is mostly the fix. I changed it from an internal state to only a memo value.

The problem was when there was an empty row on the table and after a reload the useEffect tried to remove it. Then it triggers a onChange here which changes the data which changes the table which triggers the useEffect again. Or at least something like that.
With the memo we always depend on the global state and only modify it to our needs in here.